boot/freeldr/freeldr/arch/uefi/uefimem.c

```diff
@@ -0,0 +1,293 @@
+/*
+ * PROJECT:     Freeldr UEFI Extension
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     UEFI Memory Managment Functions
+ * COPYRIGHT:   Copyright 2022 Justin Miller <justinmiller100@gmail.com>
+ */
+
+/* INCLUDES *******************************************************************/
```

Comment:
This is still too long. Column 81 currently. Strip an additional *. Same for the others.